ch_backup/logic/access.py

```diff
@@ -28,13 +33,39 @@ def backup(self, context: BackupContext) -> None:
         """
         Backup access control entities.
         """
-        objects = context.ch_ctl.get_access_control_objects()
-        context.backup_meta.set_access_control(objects)
-        access_control = context.backup_meta.access_control
 
-        if self._has_replicated_access(context):
-            self._backup_replicated(access_control.acl_ids, context)
-        self._backup_local(access_control.acl_ids, context)
+        clickhouse_access_path = context.ch_ctl_conf["access_control_path"]
+        backup_tmp_path = os.path.join(
+            context.ch_ctl_conf["tmp_path"], context.backup_meta.name
+        )
+        user = context.ch_ctl_conf["user"]
+        group = context.ch_ctl_conf["group"]
+
+        create_dir(clickhouse_access_path, user, group)
+        with temporary_directory(backup_tmp_path, user, group):
```

Comment:
1. Do we need to invent own class instead of ['TemporaryDirectory'](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)
2. Does this class produce leftovers in case of exception and is it expected

Reply:
For TemporaryDirectory there are no option to keep folder in case of exceptions.
https://github.com/python/cpython/blob/59630f92d8223f80993e3646b0f734d27f4b8dd4/Lib/tempfile.py#L944-L966

Reply:
Why do we want to leave data in a temporary directory in case of exception ? It might be corrupted

Reply:
Because it might be useful in trouble shooting. If we want to re-run operation(backup/restore), then old files will be replaced with a new one. Also it make sense to cleanup tmp folder before operation, if it exist.

Reply:
Ok, nevertheless this class is too specific for a general class(to be placed in `ch_backup/util.py`). Someone might want to accidentally reuse it for their own purposes
I see the following options:
1) Write a comment that this class is intended for using for access control backup procedure and write this motivation about keeping data.
2) Rename it to smth like `access_control_temp_directory`.
3) Create a fully functional replacement for `tempfile.TemporaryDirectory` with `keep` parameter. 
Simplified example
```
@contextmanager
def temporary_directory(keep=False):
    if keep:
        tmpdir = mkdtemp()
        print("Keeping tempdir: {}".format(tmpdir))
        yield tmpdir
    else:
        with TemporaryDirectory() as tmpdir:
            yield tmpdir
```